src/store/edit-profile/api.ts

```diff
@@ -0,0 +1,27 @@
+import { patch } from '../../lib/api/rest';
+
+export async function editUserProfile({
+  profileId,
+  name,
+  profileImage,
+}: {
+  profileId: string;
+  name: string;
+  profileImage?: string;
+}) {
+  try {
+    const response = await patch(`/api/profiles/${profileId}`).send({ profileId, firstName: name, profileImage });
+    return {
+      success: true,
+      response: response.body,
+    };
+  } catch (error: any) {
+    if (error?.response?.status === 400) {
```

Comment:
Doesn't seem like we do anything with these errors in the saga so maybe we should just not catch the error here and let it throw. Did we confirm that errors for this endpoint come back in this format? (moot point if we remove this)

Reply:
> Did we confirm that errors for this endpoint come back in this format?

We don't have specific 400 Errors for the PATCH endpoint, no. We can simply throw here.